scripts/scil_detect_streamlines_loops.py

```diff
@@ -77,36 +82,56 @@ def main():
         parser.error('Angle "{}" '.format(args.angle) +
                      'must be greater than 0')
 
-    tractogram = nib.streamlines.load(args.in_tractogram)
+    tractogram = load_tractogram(args.in_tractogram, args.reference)
     streamlines = tractogram.streamlines
 
     streamlines_c = []
-    loops = []
+    ids_c = []
+
+    streamlines_l = []
+    ids_l = []
+
     if len(streamlines) > 1:
-        streamlines_c, loops = remove_loops_and_sharp_turns(streamlines,
+        streamlines_c, ids_c = remove_loops_and_sharp_turns(streamlines,
                                                             args.angle,
                                                             args.qb,
                                                             args.threshold)
+
+        ids_l = np.setdiff1d(np.arange(len(streamlines)), ids_c)
+        streamlines_l = streamlines[ids_l]
     else:
-        parser.error('Zero or one streamline in {}'.format(args.in_tractogram) +
-                     '. The file must have more than one streamline.')
+        parser.error(
+            'Zero or one streamline in {}'.format(args.in_tractogram) +
+            '. The file must have more than one streamline.')
 
     if len(streamlines_c) > 0:
-        tractogram_c = nib.streamlines.Tractogram(streamlines_c,
-                                                  affine_to_rasmm=np.eye(4))
-        nib.streamlines.save(tractogram_c, args.out_tractogram,
-                             header=tractogram.header)
+        data_per_streamline_c, data_per_point_c = \
+            filter_tractogram_data(tractogram, ids_c)
+
+        tractogram_c = StatefulTractogram(
```

Comment:
I think we are going for a convention in regard to StatefulTractogram with sft.
so maybe sft_c ? (optional, depends on others more than me)